client/src/components/admin/AdminRound/AdminRoundContent.jsx

```diff
@@ -40,9 +40,26 @@ const ENTER_RESULTS = gql`
   ${ADMIN_ROUND_RESULT_FRAGMENT}
 `;
 
-// Persist batch results in local storage, in case people navigate
-// or refresh the page
+const IS_BATCH_MODE_KEY = `wca-live:is-batch-mode`;
 
+/**
+ * Persist whether batch mode is toggled on, in case people navigate
+ * or refresh the page
+ * @returns {boolean}
+ */
+function getStoreIsBatchMode() {
+  const json = localStorage.getItem(IS_BATCH_MODE_KEY);
+  return json ? JSON.parse(json) : false;
+}
+
+function setStoreIsBatchMode(isBatchMode) {
+  localStorage.setItem(IS_BATCH_MODE_KEY, isBatchMode);
```

Comment:
Let's add explicit conversion to JSON. The default conversion to string is equivalent for booleans, but this makes the intent clear:

```suggestion
  localStorage.setItem(IS_BATCH_MODE_KEY, JSON.stringify(isBatchMode));
```